src/server/editorServices.ts

```diff
@@ -2799,19 +2824,24 @@ namespace ts.server {
                         else {
                             // reload from the disk
                             this.reloadConfiguredProject(project, reason);
-                            // If this is solution, reload the project till the reloaded project contains the script info directly
-                            if (!project.containsScriptInfo(info) && project.isSolution()) {
-                                forEachResolvedProjectReferenceProject(
+                            // If this is project does not contain this file directly, reload the project till the reloaded project contains the script info directly

```

Comment:
Grammar ("If this is project does not contain")

Reply:
will fix this in #39613.  i pulled that out just to be able to stage things correctly.